OktaLoggerAnalytics.podspec

```diff
@@ -1,20 +1,22 @@
 Pod::Spec.new do |s|
-  s.name             = "OktaLoggerAppCenter"
-  s.version          = "1.3.7"
-  s.summary          = "Implementation of AppCenter logger destination"
-  s.description      = "Implementation of AppCenter logger destination. Requires OktaLogger/Core"
+  s.name             = "OktaLoggerAnalytics"
+  s.version          = "1.0.0"
+  s.summary          = "Implementation of Analytics logger destination"
+  s.description      = "Implementation of Analytics logger destination. Requires OktaLogger/Core"
   s.homepage         = "https://github.com/okta/okta-logger-swift"
   s.license          = { :type => 'APACHE2', :file => 'LICENSE' }
   s.author           = { "Okta Developers" => "developer@okta.com" }
   s.source           = { :git => "https://github.com/okta/okta-logger-swift.git",  :tag => s.version.to_s }
   s.osx.deployment_target = '10.14'
+  s.ios.deployment_target = '11.0'
   s.swift_version = '5.0'
   s.dependency 'SwiftLint'
   s.static_framework = true
-  
+
+  s.dependency 'AppCenter','4.3.0'
+  s.dependency 'OktaLogger/Core'
```

Comment:
Do we need explicit version, at least '~>1`? Future versions of Core may have breaking changes...